normal_operation.md

```diff
@@ -4,67 +4,46 @@ css_id: normal_operation
+### TODO Write rationale for the SHOULDs in the section above.
 
-## Backup and garbage collection for the image repository
+## Backup and garbage collection for the Image repository
 
-#### TODO Add link from last sentence in this section to the Delta update strategies subsection of the Customizations section.
+The OEM SHOULD regularly perform backup and garbage collection of the metadata and images on the Image repository. This is done to ensure the OEM is able to safely recover from a repository compromise, and that the repository continues to have sufficient storage space. To do so, an OEM MAY use either the following steps, or its own corporate backup and garbage collection policy.
 
-The OEM SHOULD regularly perform backup and garbage collection of the metadata and images on the image repository. This is done so that the OEM is able to safely recover from a repository compromise, and ensure that the repository has enough storage space. To do so, an OEM MAY use either the following steps, or its own corporate backup and garbage collection policy.
+First, an automated process SHOULD store every file on the Image repository, as well as its cryptographic hash on a separate, offline system. A copy of the inventory database from the Director repository SHOULD also be stored on this offline system. This allows administrators to detect and recover from a repository compromise. 
 
-First, an automated process SHOULD store every file on the image repository, as well as its cryptographic hash on a separate, offline system. This automated process SHOULD also store a copy of the inventory database from the director repository on this offline system. This allows administrators to detect and recover from a repository compromise.
+Second, the automated process SHOULD remove expired metadata from the image repository to reclaim storage space. If the OEM is interested in supporting delta updates for vehicles that have not been updated for a long time, then the automated process SHOULD NOT remove images associated with expired metadata, because these images MAY be needed in order to compute delta images. (See [Delta Update Strategies](https://github.com/uptane/deployment-considerations/blob/master/customizations.md#delta-update-strategies) on the Customizing Uptane page).
 
-Second, the automated process SHOULD remove expired metadata from the image repository, in order to reclaim storage space. If the OEM is interested in supporting delta updates for vehicles that have not updated for a long time, then the automated process SHOULD NOT remove images associated with expired metadata, because these images MAY be needed in order to compute delta images (see Delta Update Strategies on the Customizing Uptane page).
+### TODO Write rationale for the SHOULDs in the section above.
```

Comment:
Do we still need this? The SHOULDs seem pretty clear to me.

Reply:
This is something that Ira asked me to do so down the road we can have a formal explanation for all the should. But Patrick has basically said the same thing, so I think I will remove those comments, here and elsewhere as I finish edits. Thanks.